src/annotator/selection-observer.js

```diff
@@ -76,10 +76,10 @@ export class SelectionObserver {
 
     this._document = document_;
     this._listeners = new ListenerCollection();
-    this._events = ['mousedown', 'mouseup', 'selectionchange'];
-    for (let event of this._events) {
-      this._listeners.add(document_, event, this._eventHandler);
-    }
+
+    this._listeners.add(document_, 'selectionchange', eventHandler);
+    this._listeners.add(document_.body, 'mousedown', eventHandler);
+    this._listeners.add(document_.body, 'mouseup', eventHandler);
```

Comment:
Can you help me understand why `selectionchange` stays on the `document` here? Is there a risk that the reasoning for moving the mouse events to `document.body` could get lost for a future dev without some commenting?

Reply:
The `selectionchange` event is fundamentally associated [with the Document](https://developer.mozilla.org/en-US/docs/Web/API/Document/selectionchange_event) rather than elements.

Reply:
I'll add a comment about why these events are listened for on the body.